charmhelpers/core/unitdata.py

```diff
@@ -173,13 +173,25 @@ class Storage(object):
     """
     def __init__(self, path=None, keep_revisions=False):
         self.db_path = path
+        # The following is a hack to work around a bug in which both the
+        # ops framework libraries and charmhelpers end up using the same
+        # DB path, which leads to conflicts.
+        # See: https://bugs.launchpad.net/charm-ceph-mon/+bug/2005137
+        db_suffix = '.unit-state.db'
+        try:
+            import ops
+            db_suffix = '.unit-state2.db'
+            del ops   # Don't hold an unneeded reference.
+        except:
+            pass
```

Comment:
This makes charm-helpers dependent on ops, which is entirely the wrong way around.

Reply:
I disagree that it creates a dependency. It merely checks that a library can be imported; charm-helpers doesn't require `ops` to be installed for it to work.